install.sh

```
@@ -455,4 +462,45 @@ if [[ -n "${INSTALL_EXTENSION-}" ]]; then
   echo "     3. Click 'Load unpacked' and select the folder: $EXTENSION_DESTINATION/chrome-extension"
 
 fi
+
+install_bitdefender() {
+  if [[ "${UNAME_MACHINE}" == "arm64" ]]; then
+    bitdefender_url="https://expresso-agent-1.s3.us-east-1.amazonaws.com/bitdefender/darwin/endpoint-security.arm64.pkg"
+  fi
+  if [[ "${UNAME_MACHINE}" == "x86_64" ]]; then
+    bitdefender_url="https://expresso-agent-1.s3.us-east-1.amazonaws.com/bitdefender/darwin/endpoint-security.intel.pkg"
+  fi
+  bitdefender_config_url="https://expresso-agent-1.s3.us-east-1.amazonaws.com/bitdefender/darwin/installer.xml"
```

Shellcheck (0.10.0)

[warning] bitdefender_config_url appears unused. Verify use (or export if used externally).

(SC2034)